src/iop/borders.c

```diff
@@ -937,20 +937,23 @@ void gui_init(struct dt_iop_module_t *self)
   gtk_widget_set_tooltip_text(g->size, _("size of the border in percent of the full image"));
 
   DT_BAUHAUS_COMBOBOX_NEW_FULL(g->aspect, self, NULL, N_("aspect"),
-                               _("select the aspect ratio or right click and type your own (w:h)"),
+                               _("select the aspect ratio (right click on slider below to type your own w:h)"),
                                0, aspect_changed, self,
                                N_("image"),
                                N_("3:1"),
                                N_("95:33"),
+                               N_("Cinemascope 2.39:1"),
```

Comment:
Maybe 16:10 as well?

Reply:
Sure.

Reply:
**cinemascope** in lower case.

Reply:
CinemaScope is a proper name.  I just missed the camelcasing before...